internal/website/content/howto/pubsub/publish.md

```diff
@@ -45,7 +45,7 @@ application connects to the correct region, but otherwise `pubsub.OpenTopic`
 will use the region found in the environment variables or your AWS CLI
 configuration.
 
-{{< goexample "gocloud.dev/pubsub/awssnssqs.Example_openTopic" >}}
+{{< goexample "gocloud.dev/pubsub/awssnssqs.Example_openSNSTopicFromURL" >}}
```

Comment:
Note: I renamed the URL-based examples to have a "FromURL" suffix, not sure if that's good. I find the current examples a bit confusing:
https://godoc.org/gocloud.dev/pubsub/awssnssqs#pkg-examples

(there's no way to distinguish the URL-based examples from the other ones).

If you like this, I'll do a separate PR to update the other URL examples similarly.

Reply:
Seems fine to me: I don't have a strong preference.

Reply:
https://github.com/google/go-cloud/issues/2091